.github/workflows/pr-title-linter.yml

```diff
@@ -16,15 +16,16 @@ jobs:
     steps:
       - name: Checkout Repo
         uses: actions/checkout@v4
-
-      - name: Setup
-        id: config
-        uses: ./.github/actions/init
         with:
-          turbo-signature: ${{ secrets.TURBO_REMOTE_CACHE_SIGNATURE_KEY }}
-          turbo-team: ${{ vars.TURBO_TEAM }}
-          turbo-token: ${{ secrets.TURBO_TOKEN }}
-          playwright-enabled: true # Must be present to enable caching on branched workflows
+          fetch-depth: 1
```

Comment:
The default value for `fetch-depth` is 1. We could remove it, or leave it to be explicit.

Reply:
You're correct, the docs have the a `0` right next to the `1`, and I read that as the default.